vyper/functions/functions.py

```diff
@@ -370,8 +371,8 @@ def evaluate(self, node):
         else:
             raise UnfoldableNode
 
-        hash_ = keccak256(value)
-        return vy_ast.Bytes.from_node(node, value=hash_)
+        hash_ = f"0x{keccak256(value).hex()}"
+        return vy_ast.Hex.from_node(node, value=hash_)
```

Comment:
I had to change this back to `Hex` - `parser/expr.py` interprets a 32 byte string as `bytes[32]`. Once type checking is implemented we can try to fold out `Hex` altogether.